cmd/incus-migrate/main_migrate.go

```diff
@@ -554,11 +554,15 @@ func (c *cmdMigrate) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// Define name for converted VM image so it can be easily passed to the cleanup function
+	convertedImg := filepath.Join(path, "converted-raw-image.img")
+
 	// Automatically clean-up the temporary path on exit
-	defer func(path string) {
+	defer func(path, convertedImg string) {
```

Comment:
Tiny code style nit, but we usually always type each argument even if they're the same type.